macOS/UnitTests/DataImport/FirefoxHistoryReaderTests.swift

```diff
@@ -22,11 +22,11 @@ import Testing
 struct FirefoxHistoryReaderTests {
 
     @Test("Check if expected frecent sites are read from Firefox history database")
-    func readingFrecentSites() async throws {
+    func readingFrecentSites() throws {
         let historyReader = FirefoxHistoryReader(firefoxDataDirectoryURL: resourceURL())
         let frecentSites = try historyReader.readFrecentSites().get()
 
-        #expect(frecentSites.count == 2)
```

Comment:
I added another site (with an amazon.com URL) to the history database in the Firefox test data, for the new test below. There are now 3 “frecent” sites to read from that data.